TwitchPlaysAssembly/Src/ComponentSolvers/Modded/Misc/ResistorsComponentSolver.cs

```diff
@@ -72,15 +77,17 @@ protected internal override IEnumerator RespondToCommandInternal(string inputCom
 		if (commands.Length == 2 && commands[0].EqualsAny("hit", "press", "click"))
 			commands = commands.Skip(1).ToArray();
 
-		if (commands.Length == 1 && commands[0].EqualsAny("check", "submit"))
-		{
-			yield return null;
-			yield return DoInteractionClick(_checkButton);
-		}
-		else if (commands.Length == 1 && commands[0].EqualsAny("clear", "reset"))
+		// ReSharper disable once SwitchStatementMissingSomeCases
+		switch (commands.Length)
```

Comment:
This is a weird use of a switch statement, a `if (commands.Length == 1)` surrounding two `if (commands[0] ...` would probably be better.